.github/workflows/ci-cd.yml

```diff
@@ -7,6 +7,10 @@ on:
         description: 'Publish snapshot?'
         required: false
         default: 'false'
+      publish:
+        description: 'Publish to Sonatype Staging Repo?'
```

Comment:
Am I missing something, or why did you add this here? 🤔 

Shouldn't be in the release workflow?

Reply:
yes, that's a mistake. The publishing logic is all moved to the release.yml